org.civicrm.reqangular/src/common/services/settings/hr-settings.js

```diff
@@ -0,0 +1,12 @@
+define([
+    'common/angular',
+    'common/modules/settings'
+], function (angular, module) {
+    'use strict';
+
+    module.factory('HR_settings', function(){
```

Comment:
Aren't services in angular singletons? Doesn't it mean that in our case all the apps are going to share the same `HR_settings` object?